mkosi.profiles/desktop/mkosi.conf

```diff
@@ -3,7 +3,5 @@
 [Content]
 Packages=
         desktop-file-utils
-        flatpak
-        kdb
```

Comment:
[KDb](https://community.kde.org/KDb) seems an odd thing to install by default. Did you mean [kbd](http://www.kbd-project.org/) ?